lib/collection/proxy-config.js

```diff
@@ -52,18 +63,30 @@ _.inherit((
         ProxyConfig.super_.call(this, options);
 
         // Assign defaults before proceeding
-        _.assign(this, /** @lends ProxyConfig.prototype */ {
+        _.assign(this, /** @lends ProxyConfig */ {
             /**
              * The url mach for which the proxy has been associated with.
              * @type {String}
              */
             match: new UrlMatchPattern(),
 
             /**
-             * The server url or the proxy url.
-             * @type {Url}
+             * The proxy server host or ip
+             * @type {String}
+             */
+            host: undefined,
+
+            /**
+             * The proxy server port number
+             * @type {Integer}
              */
-            server: new Url(),
+            port: DEFAULT_PORT,
+
+            /**
+             * The protocol(s) that the proxy server supports
+             * @type {Array}
+             */
+            protocols: DEFAULT_PROTOCOLS,
```

Comment:
This will cause the `DEFAULT_PROTOCOLS` to get mutated. Do a clone of the same or `DEFAULT_PROTOCOLS.splice()`